PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -12,19 +12,23 @@ def makePuppies( process ):
 
 def makePuppiesFromMiniAOD( process, createScheduledSequence=False ):
     task = getPatAlgosToolsTask(process)
+    process.load('CommonTools.ParticleFlow.pfCHS_cff')
+    task.add(process.packedPrimaryVertexAssociationJME)
```

Comment:
Isn't better to have 
from CommonTools.ParticleFlow.pfCHS_cff import packedPrimaryVertexAssociationJME
(otherwise we load again the 
```
from CommonTools.ParticleFlow.pfNoPileUpPacked_cfi import pfNoPileUpPacked as _pfNoPileUpPacked
  pfCHS = _pfNoPileUpPacked.clone(
      vertexAssociationQuality=pfPileUpJME.vertexAssociationQuality
  )
```

(note this makePuppiesFromMiniAOD is called also on nano)